File: 백준/Bronze/1546. 평균/평균.py
if __name__ == '__main__':
  N = int(input())
  score = input()
  score = list(map(int,score.split()))
  max_score = max(score)
  for i in range(N):
    score[i] = score[i]/max_score*100
  average = sum(score) / len(score)
  print(average)

File: 백준/Bronze/1546. 평균/평균.cc
#include <iostream>
using namespace std;

int main() {
	int N;
	int score[1000];
	int max = -1;
	float total = 0;

	cin >> N;
	for (int i = 0; i < N; i++) {
		cin >> score[i];
		if (score[i] > max) {
			max = score[i];
		}
		total += score[i];
	}
	cout << total *100 / (max * N);
	return 0;
}